kstreams/engine.py

```diff
@@ -23,12 +23,14 @@ def __init__(
         *,
         consumer_class: Type[ConsumerType],
         producer_class: Type[ProducerType],
-        monitor: PrometheusMonitorType,
+        monitor: PrometheusMonitor,
         title: Optional[str] = None,
+        backend: Optional[Kafka] = None,
```

Comment:
I think `backend` should not be optional and it should be provided by the `factory` in the `create.py`